source/Calamari.AzureWebApp/Calamari.AzureWebApp.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <AssemblyName>Calamari.AzureWebApp</AssemblyName>
+    <RootNamespace>Calamari.AzureWebApp</RootNamespace>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <IsPackable>false</IsPackable>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net452</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Calamari.AzureScripting" Version="14.2.0" />
+    <PackageReference Include="Calamari.Common" Version="21.1.0" />
```

Comment:
Project Reference for Calamari.Common

Reply:
As Calamari.Common targets .netstandard on unix machines and won't be compatible for AzureWebApp, let's keep this as is for now.